SirknightMike/EcoPick
# feat(login): creating login page

### PR Description:

Created login page for the eco pick application

### QA Steps:

1. Ensure that the login page is working that registered user can login 
2. Go to the register page make sure that user can be created.

### Notes:

None
